lib/stat_tracker.rb

```diff
@@ -3,11 +3,10 @@
 require_relative 'game_stats'
 require_relative 'season_stats'
 require_relative 'team_stats'
+require_relative 'attr_readable'
```

Comment:
Got on Zoom with Eli for clarification about changes. Attr_readerable self explanitory